### PR TITLE
Add fluent interface

### PR DESCRIPTION
**Pull request by vokeit-gschuster**
Added fluent interface and fromFile() for PublicKey

### CODE REVIEW

File: src/PublicKey.php [hunk 1]
```diff
@@ -62,44 +62,68 @@ public function __construct(string $key)
...
+     * Creates a new instance from a file
+     *
+     * @throws FilePermissionException if the authorized_keys file cannot be read
+     */
+    public static function fromFile(string $file): self
+    {
+        $content = @file_get_contents($file);
+
+        if ($content === false) {
+            throw new FilePermissionException(sprintf('Could not read file "%s"', $file), 1486563469);
```
Comment (mbrodala) [inline src/PublicKey.php:75]: Please use a unique code here (current UNIX timestamp).

**Review by mbrodala — CHANGES_REQUESTED** [review]
Thanks for the suggestion, please have a look at my remarks.

**Comment by vokeit-gschuster** [comment]
Fixed.

**Comment by mbrodala** [comment]
Can you please allow editing of your fork/branch?

**Review by mbrodala — APPROVED** [review]

**Comment by mbrodala** [comment]
I squashed all commits into one and added a missing exception import. Thanks for the contribution!

This was released now: https://github.com/pagemachine/authorized-keys/releases/tag/2.2.0

**Pull request merged**